modules/admin/proc-configuring-postgresql-with-tls-using-helm.adoc

```diff
@@ -0,0 +1,146 @@
++
+[source,yaml]
+----
+cat <<EOF | oc -n <your-namespace> create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+ name: postgres-cred
+data: <1>
+  POSTGRES_PASSWORD: <password>
+  POSTGRES_PORT: "<db-port>"
+  POSTGRES_USER: <username>
+  POSTGRES_HOST: <db-host>
+  PGSSLMODE: <ssl-mode> # for TLS connection <2>
+EOF
+----
+<1> Provide credential data to connect with your PostgreSQL instance.
+<2> Provide the value based on the required link:https://www.postgresql.org/docs/15/libpq-connect.html#LIBPQ-CONNECT-SSLMODE[SSL mode].
+
+. Configure your PostgreSQL instance in the Helm configuration file named `values.yaml`:
++
+[source,yaml]
+----
+# ...
+upstream:
+ postgresql:
+   enabled: false  # disable PostgreSQL instance creation <1>
+ backstage:
+   appConfig:
+     backend:
+       database:
+          connection:  # configure Backstage DB connection parameters
+            host: ${POSTGRES_HOST}
+            port: ${POSTGRES_PORT}
+            user: ${POSTGRES_USER}
+            password: ${POSTGRES_PASSWORD}
+            ssl:
+              rejectUnauthorized: true,
+              ca:
+                $file: /opt/app-root/src/postgres-ca.pem
+              key:
+                $file: /opt/app-root/src/postgres-key.key
+              cert:
+                $file: /opt/app-root/src/postgres-crt.pem
+   extraEnvVarsSecrets:
+     - postgres-cred # inject credentials secret to Backstage <2>
+   extraEnvVars:
+     - name: BACKEND_SECRET
+       valueFrom:
+         secretKeyRef:
+           key: backend-secret
+           name: '{{ include "janus-idp.backend-secret-name" $ }}'
+   extraVolumeMounts:
+     - mountPath: /opt/app-root/src/dynamic-plugins-root
+       name: dynamic-plugins-root
+     - mountPath: /opt/app-root/src/postgres-crt.pem
+       name: postgress-external-db-cluster-cert # inject certificate secret to Backstage cont.
+       subPath: postgres-crt.pem
+     - mountPath: /opt/app-root/src/postgres-ca.pem
+       name: postgress-external-db-cluster-cert # inject certificate secret to Backstage cont.
+       subPath: postgres-ca.pem
+     - mountPath: /opt/app-root/src/postgres-key.key
+       name: postgress-external-db-cluster-cert # inject certificate secret to Backstage cont.
+       subPath: postgres-key.key              
+   extraVolumes:
+     - ephemeral:
+         volumeClaimTemplate:
+           spec:
+             accessModes:
+               - ReadWriteOnce
+             resources:
+               requests:
+                 storage: 1Gi
+       name: dynamic-plugins-root
+     - configMap:
+         defaultMode: 420
+         name: dynamic-plugins
+         optional: true
+       name: dynamic-plugins
+     - name: dynamic-plugins-npmrc
+       secret:
+         defaultMode: 420
+         optional: true
+         secretName: dynamic-plugins-npmrc
+     - name: postgres-crt
+       secret:
+         secretName: postgress-external-db-cluster-cert <3>
```

Comment:
I would also rather use `<cred-secret>` and `<crt-secret>` for Secrets as in `modules/admin/proc-configuring-postgresql-instance-using-helm.adoc` to keep the doc consistent.